specs/agents/tracing-instrumentation-messaging.md

```diff
@@ -158,7 +158,7 @@ agent should report 0 for this field.
 
 - **`context.message.body`**: similar to HTTP requests' `context.request.body`- only fill in messaging-related **transactions** (ie
 incoming messages creating a transaction) and not for outgoing messaging spans.
-   - Capture only when `ELASTIC_APM_CAPTURE_BODY` config is set to `true`.
+   - Capture only when `ELASTIC_APM_CAPTURE_BODY` config is set to `'all'` or `'transactions'`.
```

Comment:
```suggestion
   - Capture only when `ELASTIC_APM_CAPTURE_BODY` config is set to anything other than `off`.
```
Maybe a bit more accurate as it includes `errors` as well.

Reply:
Actually this should **not** capture `transaction.context.message.body` when `capture_body == 'errors'`, right?  In other words, I think my original is correct.

Reply:
Sorry, I was too haste to comment, focusing on config **type** correction without looking into the context 🙏 
You're absolutely right!